Example/ViewController.swift

```diff
@@ -540,6 +540,34 @@ extension ViewController: NavigationViewControllerDelegate {
         dismissActiveNavigationViewController()
         clearNavigationMapView()
     }
+    
+    // MARK: - Offline Navigation Tiles switching
+    
+    func tilesVersioning(wantsSwitchTilesVersion versions: [String], completion: @escaping (String?) -> ()) {
+        let alertController = UIAlertController(title: "Switch to Offline Regions",
+                                                message: "There are next tiles versions available:",
+                                                preferredStyle: .actionSheet)
+        
+        versions.forEach{ version in
+            alertController.addAction(UIAlertAction(title: version, style: .default, handler: { _ in
```

Comment:
You say `These are not designed to be displayed to User directly.` in docs. Can we somehow parse the version and provide a more human-readable string here?